docs/Support/glossary.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 2
+---
+
+# Glossary
+
+## Definitions of commonly used terms and core concepts.
+
+1. **[Public Key Cryptography](/Introduction/whatIsLitProtocol#decentralized-cryptography)** — the technology that underpins cryptocurrency and most of the security infrastructure on the web today. It allows you to do two main things:
```

Comment:
It would be helpful to have the terms in alphabetically order and with the ### heading before each one so it shows on the right hand sidebar!